[INTERNAL] Rework Documentation (#269)
SAP/ui5-tooling: scripts/resources/CLI.template.md

# UI5 CLI
## Requirements

> - [Node.js](https://nodejs.org/) version 10 or later

Reviewer: ```suggestion
- [Node.js](https://nodejs.org/) Version 10 or later
```

## Local vs. Global Installation
In general, we recommend a global installation of the UI5 CLI (`npm install --global @ui5/cli`).

> However, it makes sense to add the UI5 CLI as a [devDependency](https://docs.npmjs.com/files/package.json#devdependencies) (`npm install --save-dev @ui5/cli`) to a projects `package.json`. This not only ensures that every developer working on the project uses the same version of the UI5 CLI, your continuous integration environments will also use this version.

Reviewer: ```suggestion
However, it makes sense to add the UI5 CLI as a [devDependency](https://docs.npmjs.com/files/package.json#devdependencies) (`npm install --save-dev @ui5/cli`) to a project's `package.json`. This ensures that every developer working on the project uses the same version of the UI5 CLI and your continuous integration environments also uses this version.
```
